apps/documentation/src/routes/documentation/routing/api-routes.mdx

```diff
@@ -0,0 +1,67 @@
+import MetaTags from '@/components/MetaTags'
+
+<MetaTags
+  title="Tuono - API routes"
+  canonical="https://tuono.dev/documentation/routing/api-routes"
+/>
+
+import Breadcrumbs from '@/components/Breadcrumbs'
+
+<Breadcrumbs breadcrumbs={[{ label: 'API routes' }]} />
+
+# API routes
+
+## Overview
+
+API routes provide a solution to build a public API with Tuono.
+
+Any file inside the folder `src/routes/api` is mapped to `/api/*`
+path and will be treated as an API endpoint instead of a React route.
```

Comment:
```suggestion
Any file inside the `src/routes/api` folder is mapped to the `/api/*` path 
and will be treated as an API endpoint, rather than a React route.
```